modules/platforms/dotnet/Apache.Ignite/IgniteException.cs

```diff
@@ -77,6 +77,6 @@ public IgniteException(Guid traceId, int code, string? message, Exception? inner
         /// <summary>
         /// Gets the code as string.
         /// </summary>
-        public string CodeAsString => ErrorPrefix + '-' + GroupName + '-' + ErrorCode;
+        public string CodeAsString => ErrorPrefix + '-' + GroupName + '-' + unchecked((ushort)ErrorCode);
```

Comment:
Is it formatted like this in Java too?

Reply:
Yes. I've added `ExceptionCodeAsStringTests` to verify this.